src/components/util/ask-text-dialog.js

```diff
@@ -33,6 +33,10 @@ export const AskTextDialog = ({ title, value, show, onValidate, onClose }) => {
         setCurrentValue(e.target.value);
     };
 
+    useEffect(() => {
+        setCurrentValue(value);
```

Comment:
we mix controlled and uncontrolled value props component here... this is strange...
if value is passed as props, then we should pass a onChange callback as well then we coulad avoid this useEffect and the handleChange, and the surrentValue State.

Reply:
we do not want to change title unless we validate the dialog (but when we change node, we want to modify the default value)